### PR TITLE
Update enforcing-ssl.md

### LINKED ISSUE #15806: How to repair IIS Express cert
show

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c7ddfb2f-5dea-5e31-2a27-02a4dc16104c
* Version Independent ID: 12ca0b9a-2638-a4ce-c45f-d4579c83dafb
* Content: [Enforce HTTPS in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/security/enforcing-ssl?view=aspnetcore-3.0&tabs=visual-studio#troubleshoot-certificate-problems)
* Content Source: [aspnetcore/security/enforcing-ssl.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/security/enforcing-ssl.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-security**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #15806

### CODE REVIEW

File: aspnetcore/security/enforcing-ssl.md [hunk 1]
```diff
@@ -361,7 +361,7 @@ The Windows Subsystem for Linux (WSL) generates a HTTPS self-signed cert. To con
 
 ## Troubleshoot certificate problems
 
-This section provides help when the ASP.NET Core HTTPS development certificate has been [installed and trusted](#trust), but you still have browser warnings that the certificate is not trusted.
+This section provides help when the ASP.NET Core HTTPS development certificate has been [installed and trusted](#trust), but you still have browser warnings that the certificate is not trusted. The ASP.NET Core HTTPS development certificate that is used by [Kestrel](xref:fundamentals/servers/kestrel).
```
Comment (guardrex) [inline aspnetcore/security/enforcing-ssl.md]: I'm looking on the phone ... Hard to read a bit ... Is it missing a verb?

Comment (guardrex) [reply aspnetcore/security/enforcing-ssl.md]: Drop "that"?

Comment (Rick-Anderson) [reply aspnetcore/security/enforcing-ssl.md]: Or enjoy your evening and look tomorrow :)

**Comment by Rick-Anderson** [comment]
Do that too much and you'll need to access from the `:grave:`
 :no_mobile_phones: :no_entry_sign: :do_not_litter: 🚷  🕛 :clock12:

**Pull request merged**